cmd/fetch/fetch.go

```diff
@@ -253,8 +255,8 @@ func Command() *cobra.Command {
 	cmd.PersistentFlags().StringVar(
 		&logFile,
 		"log-file",
-		"",
-		"If set, writes to the log file specified. Otherwise, only writes to stdout.",
+		fmt.Sprintf("fetch-%s.log", time.Now().UTC().Format(time.RFC3339)),
+		"Default write it to a file with format \"fetch-{datetime}.log\". If is set with \"stdout\", write to stdout only",
```

Comment:
So is the behavior now always print to stdout and log a file by default?

Reply:
yeah, the new default is {stdout + to a file}. Open to revision here

Reply:
I think that should be fine. Just like ryan pointed out we might have really large log files but that can be a future thing.

```suggestion
		fmt.Sprintf("fetch-%s.log", time.Now().UTC().Format(time.RFC3339)),
		"By default, writes to a file named \"fetch-{datetime}.log\". If set as \"stdout\", writes will only go to stdout",
```
```suggestion
		fmt.Sprintf("fetch-%s.log", time.Now().UTC().Format(time.RFC3339)),
		"Default write it to a file with format \"fetch-{datetime}.log\". If is set with \"stdout\", write to stdout only",
```

Reply:
Oh and I think we dont need to escape the quotes here since its just output text.